packages/auth/Dockerfile

```diff
@@ -1,12 +1,13 @@
-FROM node:16.17.1-slim as builder
+FROM node:18-slim as builder
```

Comment:
Should we specify a specific version to make use of the cache? I think `18-slim` resolves to the latest version.

Reply:
`latest` is actually `19` now.
https://hub.docker.com/_/node
Do you mean `node:18.13-slim` or specifying the `sha256`?

Reply:
Yeah, I meant the latest version of `18`. I was actually thinking of something like `18.13.0-slim` to have a very specific version.

Reply:
I'm ok with that.
Can we get renovatebot to help us update it?

Reply:
https://docs.renovatebot.com/modules/manager/dockerfile/#open-feature-requests

Reply:
Looks like the guy who wanted to do the pinning feature hasn't gotten to it yet.